frontend/src/app/esri-map/esri-map.component.scss

```diff
@@ -1,12 +1,10 @@
 /* import the required JSAPI css */
-// @import 'https://js.arcgis.com/4.10/esri/css/main.css';
-@import 'https://js.arcgis.com/4.10/esri/themes/dark-green/main.css';
+@import 'https://js.arcgis.com/4.10/esri/css/main.css';
 // @import '../../../node_modules/arcgis-js-api/themes/dark/main.scss';
```

Comment:
Do we want to leave in this commented out import statement?

Reply:
Good catch, I left this in incase we want to revert back to the dark there post design review (It can be removed though)